hugr-py/src/hugr/serialization/extension.py

```diff
@@ -76,3 +77,12 @@ class Extension(ConfiguredBaseModel):
     @classmethod
     def get_version(cls) -> str:
         return get_serialisation_version()
+
+
+class Package(ConfiguredBaseModel):
+    modules: list[SerialHugr]
+    extensions: list[Extension]
```

Comment:
`extensions` is defined as _required_ in the schema.
Can we have it default to an empty list?
```python
    extensions: list[Extension] = Field(default_factory=list)
```